app/views/monitoring/correlation-id.njk

```diff
@@ -0,0 +1,52 @@
+{% extends '_layout.njk' %}
+{% from "macros/search-box.njk" import searchBox %}
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+
+{% block content %}
+  {{ govukBreadcrumbs({
+      items: [{
+        text: "Home",
+        href: "/"
+      }, {
+        text: "Monitoring",
+        href: "/monitoring"
+      }]
+    })}}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <span class="govuk-caption-l">{{ events.frn }} - {{ events.agreementNumber }} - PR{{ events.paymentRequestNumber }}</span>
+      <h1 class="govuk-heading-l">Monitoring</h1>
+
+      {% if events %}
+        <table class="govuk-table">
+          <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+              <th scope="col" class="govuk-table__header govuk-body-s">Activity</th>
+              <th scope="col" class="govuk-table__header govuk-body-s">Status</th>
+              <th scope="col" class="govuk-table__header govuk-table__header--numeric govuk-body-s">Completed</th>
+            </tr>
+          </thead>
+          <tbody class="govuk-table__body">
+            {% for event in events.events %}
+              <tr class="govuk-table__row">
+                <td class="govuk-table__cell govuk-body-s">{{ event.status.name }}</td>
+                <td class="govuk-table__cell govuk-body-s">
+                  {% if event.timestamp %}
+                    <strong class="govuk-tag govuk-tag--green">Completed</strong>
+                  {% elif events.events[loop.index0 - 1].lastEvent %}
+                    <span class="govuk-tag">In progress</span>
+                  {% else %}
+                    <strong class="govuk-tag govuk-tag--grey">Not started yet</strong>
+                  {% endif %}
+                </td>
+                <td class="govuk-table__cell govuk-table__cell--numeric govuk-body-s">{{ event.timestamp }}</td>
+              </tr>
+            {% endfor %}
+          </tbody>
+        </table>
+      {% else %}
+        <p id="no-hold-text" class="govuk-body">No events found for FRN {{events.frn}}.</p>
```

Comment:
Would whoever is using this know what FRN stands for/means? I'd assume so, but just wanted to ask the question

Reply:
Yep, they would.